### PR TITLE
Updating the documented minimum

### PR DESCRIPTION
**Pull request by chotchki**
When calling create DNS records you get an error if it is below 60secs

### CODE REVIEW

**Comment by KianNH** [comment]
image

30 seconds seems to work fine for me - do you have a screenshot of the record you was creating?

**Comment by chotchki** [comment]
I'm accessing from a Rust library, https://docs.rs/crate/cloudflare/0.9.1/source/src/endpoints/dns.rs that just passes the value through to Cloudflare.

The code snippet I'm calling:
```
self.client.request(&CreateDnsRecord {
            zone_identifier: &self.zone_id,
            params: CreateDnsRecordParams {
                ttl: Some(30), //Setting it as low as possible so ACME can see it
                name: &name,
                priority: None,
                proxied: Some(false),
                content: DnsContent::TXT {
                    content: proof_value,
                },
            },
        })?;
```

Here is a relevant output from its error logging:
```
2022-08-15T02:24:22.795570Z DEBUG reqwest::async_impl::client: response '400 Bad Request' for https://api.cloudflare.com/client/v4/zones/<snipped>/dns_records    
2022-08-15T02:24:22.797380Z DEBUG hyper::proto::h1::conn: incoming body completed
2022-08-15T02:24:22.797880Z DEBUG hyper::client::pool: pooling idle connection for ("https", api.cloudflare.com)
2022-08-15T02:24:22.800793Z ERROR hmdl::coordinator: Acme Service had an error |HTTP 400 Bad Request
1004: DNS Validation Error ({"error_chain": Array([Object({"code": Number(9021), "message": String("TTL must be between 60 and 86400 seconds, or 1 for Automatic.")})])})
messages: []
result: null
success: false
```

I think its because its a TXT record. If I try to use the web interface, Cloudflare sets the lowest TTL to 1min.

**Comment by chotchki** [comment]
Screenshot proof from the web interface:
image

**Comment by KianNH** [comment]
Which plan is that zone on? I'm thinking it might be a plan limitation.

image

**Comment by chotchki** [comment]
This is just on the free plan.

File: content/dns/manage-dns-records/reference/ttl.md [hunk 1]
```diff
@@ -24,4 +24,4 @@ It may take longer than 5 minutes for you to actually experience record changes,
 
 ## Unproxied records
 
-For **DNS only** records, you can choose a TTL between **30 seconds** and **1 day**.
\ No newline at end of file
+For **DNS only** records, you can choose a TTL between **60 seconds** and **1 day**.
```
Comment (kodster28) [inline content/dns/manage-dns-records/reference/ttl.md]:
```suggestion
For **DNS only** records, you can choose a TTL between **30 seconds** (Enterprise) or **60 seconds** (non-Enterprise) and **1 day**.
```

**Review by kodster28 — APPROVED** [review]

**Pull request merged**